src/app/components/Modals/UnbondWithdrawModal.tsx

```diff
@@ -49,12 +49,9 @@ export const UnbondWithdrawModal: React.FC<PreviewModalProps> = ({
   const delegation = delegationsAPI.find(
     (delegation) => delegation.stakingTxHashHex === txID,
   );
-  if (!delegation) {
```

Comment:
User will see a blank page in this case. I think it's exceptional situation, so to avoid this we need move this logic out of unbound modal. If no delegation is found pls display error modal instead @jeremy-babylonlabs @jrwbabylonlab